apps/coscrad-frontend/src/app/App.tsx

```diff
@@ -1,20 +1,24 @@
-import { Route, Routes } from "react-router-dom";
+import { Route, Routes } from 'react-router-dom';
 import About from '../components/About/About';
-import AllEntities from '../components/AllEntities/AllEntities';
+import { AllResources } from '../components/AllResources/AllResources';
+import { DynamicResourceDetailPage } from '../components/dynamicViews/dynamicResourceDetailView/DynamicResourceDetailPage';
+import DynamicIndexPage from '../components/dynamicViews/dynamicResourceIndexView/DynamicIndexPage';
 import Home from '../components/Home/Home';
 import MembersOnly from '../components/MembersOnly/MembersOnly';
 
 export function App() {
-  return (
-    <div>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path='About' element={<About />} />
-        <Route path='AllEntities' element={<AllEntities />} />
-        <Route path='MembersOnly' element={<MembersOnly />}/>
-      </Routes>
-    </div>
-  );
+    return (
+        <div>
+            <Routes>
```

Comment:
BrowserRouter must be at the Main.tsx?

Reply:
That's right. Am I breaking convention here?